develop/GMapHeatmap.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  compose,
+  defaultProps,
+  withHandlers,
+  withState,
+  withContext,
+  withProps,
+  withPropsOnChange,
+} from 'recompose';
+import { createSelector } from 'reselect';
+
+import withStateSelector from './utils/withStateSelector';
+import ptInBounds from './utils/ptInBounds';
+import GoogleMapReact from '../src';
+import SimpleMarker from './markers/SimpleMarker';
+import { susolvkaCoords, generateMarkers, heatmapData } from './data/fakeData';
+
+export const gMapHeatmap = (
+  {
+    style,
+    hoverDistance,
+    options,
+    mapParams: { center, zoom },
+    onChange,
+    onChildMouseEnter,
+    onChildMouseLeave,
+    markers,
+    draggable, // hoveredMarkerId,
+  }
+) => (
+  <GoogleMapReact
+    draggable={draggable}
+    style={style}
+    options={options}
+    hoverDistance={hoverDistance}
+    center={center}
+    zoom={zoom}
+    onChange={onChange}
+    onChildMouseEnter={onChildMouseEnter}
+    onChildMouseLeave={onChildMouseLeave}
+    resetBoundsOnResize={true}
+    apiKey={'AIzaSyC-BebC7ChnHPzxQm7DAHYFMCqR5H3Jlps'}
```

Comment:
@ZAKdev What is this apiKey here for?

Reply:
@itsmichaeldiego, we can remove it, its not doing anything.

Reply:
@itsmichaeldiego, i removed.

Reply:
@ZAKdev If I am not mistaken I believe there are a few more lines that could be removed

Reply:
@itsmichaeldiego i removed unnecessary props

Reply:
Great dude! Thanks